README.md

```diff
@@ -45,145 +62,174 @@ pip install eqlpy
 
 You can find the latest version on the [Python Package Index (PyPI)](https://pypi.org/project/eqlpy).
 
-## Usage
+## Usage with Django
+
+### Defining an encrypted field
 
-### Importing the package
+Let's say you have a customer model with an encrypted `name` field.
+You can use the `EncryptedText` type when defining the field in the model:
 
 ```python
-from eqlpy import (
-    EqlInt,
-    EqlBool,
-    EqlDate,
-    EqlFloat,
-    EqlText,
-    EqlJsonb,
-    EqlRow
-)
+from django.db import models
+from eqlpy.eqldjango import *
+
+class Customer(models.Model):
+    name = EncryptedText()
+```
+
+The above customer model would create a database table like this:
+
+```sql
+CREATE TABLE myapp_customer (
+    "id" bigint NOT NULL PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    "name" cs_encrypted_v1 NOT NULL,
+);
 ```
 
-### EQL value classes
+[!NOTE]
```

Comment:
This prints as literal string. Maybe it's some other Markdown's syntax not supported by GitHub?

Reply:
Yeah this isn't [using the GitHub Markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) syntax correctly. 

It should be: 

```suggestion
> [!NOTE]
> The column type in the database is `cs_encrypted_v1` which is encrypted type defined in [EQL](https://github.com/cipherstash/encrypt-query-language/blob/main/docs/reference/PAYLOAD.md). Internally EQL uses JSON.
```